onnx/defs/math/defs.cc

```diff
@@ -538,13 +554,21 @@ output data (Tensor<T>) where the function `f(x) = slope * x for x < 0`,
 `f(x) = x for x >= 0`., is applied to the data tensor elementwise.
 )DOC";
 
+static std::string GetPReluVer9Doc() {
+#ifndef __ONNX_NO_DOC_STRINGS
```

Comment:
suggestion: create a macro for this pattern, so that all later added ones can reuse the pattern and we don't need to revisit this issue any more.

Reply:
Currently there are 3 ways SetDoc is accomplished.
* [This](https://github.com/onnx/onnx/blob/master/onnx/defs/generator/defs.cc#L19) is easy to eliminate since it is taken care of in the SetDoc function itself. 
* [This](https://github.com/onnx/onnx/blob/master/onnx/defs/logical/defs.cc#L175) can be hidden behind a generic macro as follows:
```
.SetDoc(GET_OP_DOC_STR(std::string(BitShift_ver11_doc) + GenerateBroadcastingDocMul())

#ifndef __ONNX_NO_DOC_STRINGS
#define GET_OP_DOC_STR(doc_str) (doc_str)
#else
#define GET_OP_DOC_STR(doc_str) ("")
#endif
```
* Finally the third way is [this](https://github.com/onnx/onnx/blob/master/onnx/defs/logical/defs.cc#L28). This is a bit tricky to do in a generic way due to the way doc is constructed. Having the ```ifndef __NO_DOC_STRINGS``` inside SetDoc is not enough in this case. Surrounding SetDoc with this macro also doesn't work since it's too late. Once the 'doc' variable is used, the linker can't throw it away in the generated binary. I don't have a good solution for this one right now. Some more thought might be required.

I can change the 2nd one behind a generic macro.